### DISCUSSION TITLE
Problem with displaying upcoming meetings in the footer section

### DISCUSSION THREAD

**Discussion by Rhona-Georgia** [code4recovery/12-step-meeting-list#1585] [Q&A]
Hi all,

My URL is georgiaca.org
On the footer of each page I have displayed upcoming meetings. There is now an issue with it showing the following (see screen shot)
![12-15-2024 5-09-43 PM](https://github.com/user-attachments/assets/9458d1b9-9890-4298-84d6-a727ff60ba6c)
:
How do I go about fixing this? Thank you!

Rhona

**Comment by joshreisner** [comment]
thanks for the report @Rhona-Georgia - working on a fix!

**Reply by joshreisner** [reply]
ok, `3.16.6` has been released, hopefully fixing this issue for you…

**Reply by Rhona-Georgia** [reply]
Hi John,

Thank you so much for your quick response. I'm actually showing 3.16.7 as the version now. Everything is working correctly!

Rhona
